app/helpers/patient_query_helper.rb

```diff
@@ -74,11 +75,17 @@ def validate_patients_query(unsanitized_query)
       raise InvalidQueryError.new(:tz_offset, tz_offset) unless tz_offset.to_i.to_s == tz_offset.to_s
 
       query[:filter] = unsanitized_query[:filter].collect do |filter|
-        permitted_filter_params = filter.permit(:value, :numberOption, :dateOption, :relativeOption, :additionalFilterOption, filterOption: {}, value: {})
+        permitted_filter_params = if filter[:value].is_a? Array
+                                    filter.permit(:value, :numberOption, :dateOption, :relativeOption, :additionalFilterOption,
+                                                  filterOption: {}, value: %i[label value])
+                                  else
+                                    filter.permit(:value, :numberOption, :dateOption, :relativeOption, :additionalFilterOption,
+                                                  filterOption: {}, value: {})
+                                  end
         {
           filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :tooltip,
                                                              options: [], fields: [:name, :title, :type, { options: [] }]),
-          value: permitted_filter_params[:value] || filter.require(:value) || false,
+          value: permitted_filter_params[:value] || filter.permit(:value)[:value] || false,
```

Comment:
I think it might have to be this instead because the value of `params.require(:value)` is the same as `params.permit(:value)[:value]`

```suggestion
          value: permitted_filter_params[:value] || permitted_filter_params[:value][:value] || false,
```

Reply:
I don't think `permitted_filter_params[:value][:value]` works because `permitted_filter_params[:value]` is an Array. Should I change it back to `filter.permit(:value)[:value]` since we now allow `value` to be an array? This would mean I would have to change how we handle the filter in `advanced_filter`

Reply:
hmm yeah this is a bit tricky because `value` could be multiple data types, would it be feasible to just make a new param called `values` for when `value` is an array instead?

Reply:
Would this be a change made on the frontend?

Reply:
hmm, @sgober what do you think?

Reply:
I think that would be a pretty hefty change front end given that value can be a string, date, number, null, array, object or and array of objects...

Reply:
that's fair, I think we can just do this then to reduce redundancy
```suggestion
          value: permitted_filter_params[:value] || false,
```